docs/team/nicollegann.md

```diff
@@ -6,22 +6,63 @@ title: Nicolle's Project Portfolio Page
 ### Project: Harmonia
 
 **Overview**
-Harmonia is a desktop app for students to manage their academic and non-academic tasks and events.
+
+Harmonia is a command-line-based personal task manager, which helps university students to manage their academic tasks. It is written in Java and has a GUI implemented with JavaFX.
 
 Given below are my contributions to the project.
 
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=nicollegann&breakdown=true)
 
-* **Enhancements to implemented**: To be added
 
-* **Contributions to the UG**: To be added
+* **Enhancements implemented**:
+
+  * Evolved AddressBook into Harmonia 
+    * Modified `EditCommand` and `EditCommandParser` by replacing the fields in AB3 (phone, email, address) with fields used in Harmonia (description, deadline) (PR: [#125](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/125)).
+    * Modified `FindCommand` and `ClearCommand` to suit Harmonia (PR: [#121](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/121)).
+    * Replaced sample data for AB3 with sample data for Harmonia.
+    * Modified test utilities (PR: [#130](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/130)).
+    * Modified tests for `edit`, `find` and `clear` commands (PR: [#137](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/137)).
+    * Refactored test for `LogicManager`
+
+  * Enhancement to the `find` feature (PR: [#156](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/156))
+    * Added support to search by tags.
+
+  * Enhancement to the `help` feature (PR: [#188](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/188))
+    * Added the option of opening the user guide using the user's default browser.
+    * Justification: This enhancement provides convenience for the user as they can choose to view the user guide by clicking the button instead of having to copy and paste the url into another browser window.
```

Comment:
Should this be "This enhancement provides convenience for the user as they can choose to view the user guide by clicking the button instead of copying and pasting the url into another browser window."?